### DISCUSSION TITLE
/toggle-settings-in-the-property-pane-of-your-sharepoint-framework-web-part/

### DISCUSSION THREAD

**Discussion by comment-migrator** [estruyf/eliostruyf-comments#233] [Comments]
Toggle settings in the property pane of your SharePoint Framework web part
        
[https://www.eliostruyf.com/toggle-settings-in-the-property-pane-of-your-sharepoint-framework-web-part/](https://www.eliostruyf.com/toggle-settings-in-the-property-pane-of-your-sharepoint-framework-web-part/)

**Comment by comment-migrator** [comment]
```header
Originally posted by Anoop on 2016-10-09 10:29
```

This is brilliant! I like it. Used this functionality for PropertyPaneChoiceGroup.

**Comment by comment-migrator** [comment]
```header
Originally posted by Zllzn on 2017-03-27 15:48
```

It seems that the prototype of the function has changed :

protected getPropertyPaneConfiguration(): IPropertyPaneConfiguration

Also, it does not seem to work anymore ? any clue ? \[EDIT\] : Works very well, sorry

**Reply by comment-migrator** [reply]
```header
Originally posted on 2017-03-30 08:46
```

Indeed, some internal methods changed. I updated the code snippet with the new names. Code should still work the same way. What problems do you see? Any errors?

**Reply by comment-migrator** [reply]
```header
Originally posted by Zllzn on 2017-03-30 09:41
```

My bad, it was my validation function that made the property pane crash !  
I just retested with a new web part and only your code, works like a charm !

**Comment by princebagthariya** [comment]
How to set default of it ???
